compiler/daml-lf-ast/src/DA/Daml/LF/Ast/World.hs

```diff
@@ -21,14 +32,11 @@ import DA.Daml.LF.Ast.Version
 -- the modules of the current package. The latter shall always be closed under
 -- module dependencies but we don't enforce this here.
 data World = World
-  { worldImported :: HMS.HashMap PackageId Package
-  , worldSelf :: Package
+  { _worldImported :: HMS.HashMap PackageId Package
+  , _worldSelf :: Package
```

Comment:
Why prefix them with an underscore? I'm not a big fan of this. IIRC, all the other field names in the DAML-LF don't start with an underscore but the lenses do instead. I think we should keep this convention.

Reply:
Because you never refer to `worldImported` which makes it an unused top-level definition. Ugly, and I think a flaw in GHC's warnings personally, but this was the most practical way of solving it.

FWIW I hate both underscore prefixing of fields _and_ lenses...